transpiler/src/ast/core/intermediate-ast/nodes/package/packageAdapters/PackageAdapterListNode.ts

```diff
@@ -0,0 +1,11 @@
+import { BitloopsTypesMapping } from '../../../../../../helpers/mappings.js';
+import { IntermediateASTIdentifierNode } from '../../IntermediateASTIdentifierNode.js';
+import { TNodeMetadata } from '../../IntermediateASTNode.js';
+
```

Comment:
`IntermediateASTIdentifierNode` should be `IntermediateASTNode`

Reply:
True, thanks!